app/res/values/strings.xml

```diff
@@ -840,4 +842,12 @@
     <string name="connect_payment_dialog_no">No</string>
     <string name="connect_payment_dialog_yes">Yes</string>
     <string name="connect_payment_revert">Revert</string>
+
+    <string name="connect_deactivate_account">Deactivate Account</string>
+    <string name="connect_deactivate_account_message">You are about to permanently deactivate this account…</string>
+    <string name="connect_deactivate_account_delete">Delete Account</string>
+    <string name="connect_deactivate_account_go_back">Go Back</string>
+    <string name="connect_deactivate_account_deactivated">Account deactivated, go create a new one</string>
```

Comment:
Change to "Account deactivated, you may now register a new account with your primary phone number"

Reply:
Done